templates/desktop/developers.html

```diff
@@ -77,18 +77,16 @@
     <div class="row">
       <div class="col-8">
         <h2>What&rsquo;s new in Ubuntu {{latest_release_full}}?</h2>
-        <p class="eight-col">Ubuntu {{latest_release_full}}  brings a richer and simplified user experience with easier printing, a redesigned productivity application and a number of application updates.</p>
+        <p class="eight-col">Ubuntu {{latest_release_full}} brings a new desktop environment to Ubuntu with a redesigned interface. It has been designed to get you up and running quickly and to help keep you focused on the task at hand.</p>
       </div>
     </div>
     <div class="row">
       <div class="col-12">
         <ul class="p-list--divided is-split">
-          <li class="p-list__item is-ticked">LibreOffice 5.3 with new UI for improved usability</li>
-          <li class="p-list__item is-ticked">Print without printer-specific drivers on selected printers</li>
-          <li class="p-list__item is-ticked">Most GNOME applications updated to 3.24</li>
-          <li class="p-list__item is-ticked">Linux 4.10 kernel</li>
-          <li class="p-list__item is-ticked">New installations use swap file instead of swap partition</li>
-          <li class="p-list__item is-ticked">Fast install snaps with snap URL Support added to Ubuntu Software 3.22</li>
+          <li class="p-list__item is-ticked">LibreOffice 5.4</li>
+          <li class="p-list__item is-ticked">Print without printer-specific drivers on AirPrint, IPP everywhere, Mopria and Wifi Direct printers</li>
+          <li class="p-list__item is-ticked">A new suite of GNOME desktop applications, running the bang up to date 3.26 release.  As well as the latest the latest browsers, an email client</li>
+          <li class="p-list__item is-ticked">Linux 4.13 kernel</li>
```

Comment:
Swap this with the <li> above it so that the one-line bullets line up and look a little better. Make sure that you update the copydoc too